packages/bot-api/src/main/Bot.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {APIClient} from '@wireapp/api-client';
+import {ClientType} from '@wireapp/api-client/dist/commonjs/client/';
+import {Config} from '@wireapp/api-client/dist/commonjs/Config';
+import {Connection, ConnectionStatus} from '@wireapp/api-client/dist/commonjs/connection';
```

Comment:
Missing `/` at the end

Reply:
I will add it but since `forceConsistentCasingInFileNames` is turned on in our `tsconfig.json` file we don't need to pay attention to it anymore. 😃